scripts/validate-gentx.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+JUNOD_HOME="/tmp/junod$(date +%s)"
+RANDOM_KEY="randomjunodvalidatorkey"
+CHAIN_ID=juno-1
+DENOM=ujuno
+# MAXBOND=90000000 # 90JUNO
+
+GENTX_FILE=$(find ./$CHAIN_ID/gentx -iname "*.json")
+LEN_GENTX=$(echo ${#GENTX_FILE})
+
+# Gentx Start date
+start="2021-09-01 01:00:00Z"
+# Compute the seconds since epoch for start date
+stTime=$(date --date="$start" +%s)
+
+# Gentx End date
+end="2021-09-08 10:00:00Z"
+# Compute the seconds since epoch for end date
+endTime=$(date --date="$end" +%s)
+
+# Current date
+current=$(date +%Y-%m-%d\ %H:%M:%S)
+# Compute the seconds since epoch for current date
+curTime=$(date --date="$current" +%s)
+
+if [[ $curTime < $stTime ]]; then
+    echo "start=$stTime:curent=$curTime:endTime=$endTime"
+    echo "Gentx submission is not open yet. Please close the PR and raise a new PR after 30-August-2021 10:00:00 UTC"
+    exit 0
+else
+    if [[ $curTime > $endTime ]]; then
+        echo "start=$stTime:curent=$curTime:endTime=$endTime"
+        echo "Gentx submission is closed"
+        exit 0
+    else
+        echo "Gentx is now open"
+        echo "start=$stTime:curent=$curTime:endTime=$endTime"
+    fi
+fi
+
+if [ $LEN_GENTX -eq 0 ]; then
+    echo "No new gentx file found."
+    exit 0
+else
+    set -e
+
+    echo "GentxFiles::::"
+    echo $GENTX_FILE
+
+    echo "...........Init Juno.............."
+
+    git clone https://github.com/CosmosContracts/Juno
+    cd Juno
+    git checkout hera
```

Comment:
do we need to create new tag for mainnet? after merge https://github.com/CosmosContracts/juno/pull/53